modules/hosted-che/partials/proc_creating-a-workspace-from-template-in-hosted-che.adoc

```diff
@@ -17,15 +17,7 @@ Various programming languages and frameworks are supported.
 
 . Navigate to the link:https://www.eclipse.org/che/getting-started/cloud/[templates registry].
```

Comment:
Navigate to the link:<space>https://www.eclipse.org/che/getting-started/cloud/[templates registry].

Reply:
this is part of the adoc syntax
please check https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/